vanilla_first_setup/views/hostname.py

```diff
@@ -69,7 +69,7 @@ def __validate_hostname(self, hostname):
         if len(hostname) > 64:
```

Comment:
The max length of a hostname is 253 characters, the 64 limit only applies to each separate label

Reply:
Multiple labels are not recommended:
https://www.man7.org/linux//man-pages/man5/hostname.5.html

Reply:
Then the dot shouldn't be a valid character in the hostname

Reply:
Though even if labels aren't recommended, the max length of a hostname is 253 characters, not 64

Reply:
That's correct.
Do you want to remove that in this PR @jardon ?
Otherwise I can open a separate one myself.

Reply:
> Though even if labels aren't recommended, the max length of a hostname is 253 characters, not 64

I don't think we should allow hostnames that are not recommended in an onboarding app.
Users that are technical enough to require a non-recommended hostname will know how to change it manually.

Reply:
Right I misunderstood something, you're right